articleEditor/editor.js

```diff
@@ -0,0 +1,346 @@
+(function () {
+    window.page_content_wrapper = document.querySelector(`.page-content-wrapper`),
+        actionForm = document.querySelector(`[data-action]`);
+    let loginWindow = `../registration/start.html`;
+
+    window.articlesFromLS = localStorage.getItem(`Articles`) ? JSON.parse(localStorage.getItem(`Articles`)) : [];
+
+
+    window.articlesCounter = counter();
+
+    console.log(window.articlesFromLS);
+    authData.find(x => {
+        if (x.isLogin) {
+            window.nickName = x.sNickName;
+        }
+    });
+
+    ShowArticles(window.articlesFromLS);
+    CarouselMaket(window.articlesFromLS);
+    reattachListeners();
+
+    ["closeContent", "addContent"].map(x => document.querySelector(`.${x}`).addEventListener(`click`, () => {
+        inputValues();
+        ["show"].map(v => window.page_content_wrapper.classList.toggle(v));
+        setTimeout(() => {
+            document.querySelector("footer").scrollIntoView({behavior: "smooth", block: "end", inline: "nearest"});
+        },10);
+        window.actionForm.setAttribute(`data-action`, `save`);
+
+    }));
+
+    document.getElementById('createContent').addEventListener(`click`, () => {
+        let action = window.actionForm.getAttribute(`data-action`);
+        switch (action) {
+            case `save` :
+                document.querySelector("footer").scrollIntoView({behavior: "smooth", block: "end", inline: "nearest"});
+                createCard(checkValuesArticle());
+                document.getElementById(`SearchArticle`).value = "";
+                localStorage.setItem(`articlesCounter`, articlesCounter());
+                addArticleToStorage(window.valuesChecked);
+                ShowArticles(window.articlesFromLS);
+                CarouselMaket(window.articlesFromLS);
+                CheckIfHiddenCardsContent();
+                reattachListeners();
+                warning("");
+                inputValues();
+                window.page_content_wrapper.classList.toggle("show");
+                break;
+
+            case `edit` :
+                window.articlesFromLS.filter(article => {
+                    if (article.articleId === window.id) {
```

Comment:
Do not use `window` global object like this.
As I tell you before, use it as global container. Ex.:
`window.myEditorProject.articles = [{ name: ..., id: ..., }]`
In `window.myEditorProject` you should insert all other objects, like `page_content_wrapper` and etc.